### DISCUSSION TITLE
Llama model load problem

### DISCUSSION THREAD

**Discussion by zheng48** [ggml-org/llama.cpp#9032] [Q&A] [answered]
I encountered some load model problems. I am using llama-cli on aws ubuntu22.04 and want to load the llama model. My model is stored in the models folder. No matter whether I use llama 3.1 8b 2q_k.gguf or llama 2 7b 2q_k.gguf, it cannot load the model and display the following question,

![llama_load_problem01](https://github.com/user-attachments/assets/67b07347-62a9-475e-a82d-9bca1eef5704)

Is my command error or models problem? thx

**Comment by ngxson** [comment, marked as answer]
Your model file is corrupted (i.e. the download link is incorrect), you must re-download it.

Also try `ls -la`, you will see your current model is only 1KB
